ui/components/app/transaction-icon/transaction-icon.js

```diff
@@ -63,6 +63,8 @@ TransactionIcon.propTypes = {
     TRANSACTION_STATUSES.REJECTED,
     TRANSACTION_GROUP_STATUSES.CANCELLED,
     TRANSACTION_STATUSES.DROPPED,
+    TRANSACTION_STATUSES.CONFIRMED,
```

Comment:
these statuses are not dealt with and default to the OK_COLOR which is fine for confirmed, but needs to be PENDING_COLOR for submitted. I'm also unsure why we have a mix of individual statuses and group statuses in this map but that's a story for a different day. If we just add SUBMITTED: PENDING_COLOR to the status color map that'll be sufficient for now.